JMSPerfHarness/.gitignore

```diff
@@ -1,2 +1,4 @@
 perfharness.jar
 VERSION.txt
+.gitignore
+/bin/
```

Comment:
This doesn't seem to have worked... there are class files included in the change request within the bin directories, this may be due to the leading '/'